cmd/podman/create.go

```diff
@@ -626,19 +632,37 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime, imageName string,
 		return nil, errors.Errorf("cannot pass additional search domains when also specifying '.'")
 	}
 
+	ImageVolumes := []string{}
+	for k := range data.ContainerConfig.Volumes {
+		ImageVolumes = append(ImageVolumes, k)
+	}
+
+	var imageVolType = map[string]string{
+		"bind":   "",
+		"tmpfs":  "",
+		"ignore": "",
+	}
+	if c.IsSet("image-volume") {
```

Comment:
If you default image-volume to bind, then you don't need this check.

Reply:
fixed